Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUDeviceImpl.cpp

```diff
@@ -474,7 +481,7 @@ static auto convertToBacking(const RenderPipelineDescriptor& descriptor, Convert
             static_cast<uint32_t>(backingBuffers.size()),
             backingBuffers.data(),
         }, {
-            nullptr,
+            descriptor.primitive && descriptor.primitive->unclippedDepth ? &depthClipControl.chain : nullptr,
```

Comment:
We should only do this if the clip control feature is enabled.

Reply:
Good catch! I just discovered WebGPU requires opt-in for features

Reply:
I will add this validation to `Device::createRenderPipeline` so we return an invalid pipeline when the feature is not enabled and `unclippedDepth == true` which is what I believe the specification requires